PowerPointLabs/PowerPointLabs/AudioMisc/Audio.cs

```diff
@@ -17,7 +17,11 @@ public enum AudioType
         public const int GeneratedSamplingRate = 22050;
         public const int RecordedSamplingRate = 11025;
         public const int GeneratedBitRate = 16;
-        public const int RecordedBitRate = 8;
+        public const int RecordedBitRate = 16;
+        public const int GeneratedChannels = 1;
+        public const int RecordedChannels = 1;
+        public const string RecordedFormatName = "WAVE";
+        public const string RecordedFormatExtension = "wav";

```

Comment:
@damithc According to the Naming Conventions, Constant fields use Pascal casing (first letter of every word is capitalized)